client/js/lounge.js

```diff
@@ -205,6 +207,7 @@ $(function() {
 			"action",
 		].indexOf(type) !== -1) {
 			switch (type) {
+			case "invite": data.msg.formattedAction = "invited " + data.msg.target + " to"; break;
```

Comment:
You're not misunderstanding. This is merely future-proofing. Specifically http://ircv3.net/specs/extensions/invite-notify-3.2.html